src/pages/registration/register.page.ts

```diff
@@ -89,18 +100,21 @@ export class RegisterPage implements OnInit {
       if (err.frontendErrorCode === 'SERVER_ERROR') {
         throw 'API endpoint error';
       }
-      let message = this.registrationErrMessage + `${supportEmail}`;
+      let message = this.registrationErrMessage + supportEmail;
       if (err && err.data && err.data.msg) {
         switch (err.data.msg) {
           case 'Invalid user':
-            message = this.invalidUserErrMessage + `${supportEmail}`;
+            message = this.invalidUserErrMessage + supportEmail;
           break;
           case 'No password':
             message = this.noPasswordErrMessage;
           break;
           case 'User already registered':
             message = this.registeredErrMessage;
           break;
+          default:
+            message = this.registrationErrMessage + supportEmail;
```

Comment:
@DamengRandom 
1) on line 103, we have one similar code
```javascript
let message = this.registrationErrMessage + supportEmail;
```

we should choose to keep either of them (even though it doesn't cause error), but shouldn't both of it and the following:
```javascript
default:
    message = this.registrationErrMessage + supportEmail;
break;
```

2) when you do the following, you are indirectly *telling other coder to __remember__ adding a space* at the end of string for _this.registrationErrMessage_, human tend to be forgetful
```javascript
// or it will become "registration error messagesupport email"
console.log(this.registrationErrMessage + supportEmail); 

// it will be better if you utilise benefit of ES6
console.log(`${this.registrationErrMessage} ${supportEmail}`); // "registration error message support email"
```

Reply:
Thx @trtshen, I will change it now. ^_^